portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<div id="niceback">
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Michael's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<div id="content">
+      <h1><p3 style="color:white">Michael Peters's Portfolio </p3></h1>
+      <center><img src="/images/IMG_3549.jpg" width="120" height="190"></center>
+      
+      
+      
+
+<div id="greeting-container"></div>
     </div>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>About Me</p4>
+    </div id="Me">
+    <p1 style="color:white">My name is Michael Peters. I was born and raised in Southern California. 
+          I was raised alongside my triplet siblings, me included, and my older brother.
+        What got me into the major of Computer Information Technology was the fact that 
+        Since I was kid, i was told that going into the medical field would gurantee success.
+        However, it wasn't my main interest. It was technology that would captrue my attention.
+        Whether it was gaming or helping my family with their devices by troubleshooting and finding
+        the issue, it was always my niche.
+    </p1>
+
+    <div>
+        <p1 style="color:white"> I am now attending the California State University of Northridge and am pursuing a
+        degree in Computer Information Technology and a minor in Geographic Information Science.
+        In the future with my degree, I hope to become a Data Scientist. Another occupation that 
+        I am also highly interested in becoming is a Cyber Security Analyst.
+        </p1>
+    </div>
+
+    
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My LinkedIn and GitHub</p4>
+    </div>
+    <div><p1 style="color:white"> https://github.com/mike39608/Coding-Portfolio </p1></div>
+    <div><p1 style="color:white"> https://www.linkedin.com/in/michael-peters-4686b1227/ </p1></div>
```

Comment:
Per the [MDM documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), the `div` tag does not inherently represent something or have some semantic meaning. It's used to group content / apply some css class / id across multiple subcomponents. However, this specific `div` tag only has one element, so it can be removed (styling a group of 1 is the same thing as styling the thing itself).